libbeat/docs/dashboards.asciidoc

```diff
@@ -12,57 +12,45 @@
 
 
 [[load-kibana-dashboards]]
-==== Importing the Dashboards
+==== Set up the dashboards
 
-{beatname_uc} comes packaged with the `scripts/import_dashboards` script that you can use to import the example dashboards,
-visualizations, and searches for {beatname_uc}. The script also creates an index pattern,
-+{beatname_lc}-*+, for {beatname_uc}.
+{beatname_uc} comes packaged with example Kibana dashboards, visualizations,
+and searches for visualizing {beatname_uc} data in Kibana. Before you can use
+the dashboards, you need to run the `setup` command to create the index
+pattern, +{beatname_lc}-*+, and load the dashboards into the Kibana index.
 
-The steps in this section show how to import {beatname_uc} dashboards. You may want to import dashboards for more than
-one Beat or specify import options that aren't described here. See {beatsdevguide}/import-dashboards.html[Importing Existing Beat Dashboards]
-in the _Beats Platform Reference_ for a full list of command-line options.
+NOTE: Starting with Beats 6.0.0, the dashboards are loaded via the Kibana API.
+This requires a Kibana endpoint configuration. You should have configured the
+endpoint in <<{beatname_lc}-configuration>>. If you did't, configure it now
```

Comment:
A restart isn't required because the config will be read when setup is run. Also there's a typo on "didn't".